app/models/passenger.rb

```diff
@@ -0,0 +1,27 @@
+class Passenger < ApplicationRecord
+  has_many :trips, dependent: :destroy
+  has_many :drivers, through: :trips
+
+  validates :name, presence: true
+  validates :name, format: { with: /[\w]+([\s]+[\w]+){1}+/, message: "bad format" }
+  validates :phone_num, presence: true
+  validates :phone_num, format: { with: /\d{3}-\d{3}-\d{4}/, message: "bad format" }
+
+  def total_cost
+    total = 0
+    self.trips.each do |trip|
+      total += (trip.cost/100.to_f)
+    end
+    return total
```

Comment:
You might want to look into the Enumerables `sum` method.

You could add them all up and then divide by 100 after.